playbooks/provisioner/openstack/plugins.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Setup BMC
+  sudo: yes
+  hosts: bmc
+  roles:
+      - { role: ovb/bmc, when: provisioner.nodes.bmc is defined }
```

Comment:
please put this under "provisioner/openstack/bmc" since this is Openstack specific and there is no reason to have it as a standalone role